public/js/components/Page.js

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
+import React, { Component } from 'react';
```

Comment:
yep, looks like you always need React because JSX uses it: https://stackoverflow.com/questions/44141010/do-i-need-to-import-react-for-stateless-functional-components/44141043